core/src/main/java/org/apache/struts2/config/StrutsXmlConfigurationProvider.java

```diff
@@ -54,6 +54,7 @@ public class StrutsXmlConfigurationProvider extends XmlConfigurationProvider {
         put("-//Apache Software Foundation//DTD Struts Configuration 2.3//EN", "struts-2.3.dtd");
         put("-//Apache Software Foundation//DTD Struts Configuration 2.5//EN", "struts-2.5.dtd");
         put("-//Apache Software Foundation//DTD Struts Configuration 6.0//EN", "struts-6.0.dtd");
+        put("-//Apache Software Foundation//DTD Struts Configuration 6.5.0//EN", "struts-6.5.0.dtd");
```

Comment:
I would keep two digits pattern as such changes cannot be introduced with `PATCH` version anyway. So instead of 
```java
put("-//Apache Software Foundation//DTD Struts Configuration 6.5.0//EN", "struts-6.5.0.dtd")
```
use
```java
put("-//Apache Software Foundation//DTD Struts Configuration 6.5//EN", "struts-6.5.dtd")
```
and rename the corresponding file.

---

What do you think? does it make sense?

Reply:
Yeah. It make sense. I've updated the naming accordingly. Thanks.